src/tests/ftest/io/posix_stat.py

```diff
@@ -0,0 +1,121 @@
+            # created by ior is near the time of the start of the ior command
+            # execution.
+            current_epoch = -1
+            cmd_output = ""
+            # Run date command in the client node because that's where the file
+            # is created.
+            task = run_task(self.hostlist_clients, "date +%s")
+            for output, _ in task.iter_buffers():
+                cmd_output = "\n".join(
+                    [line.decode("utf-8") for line in output])
+            current_epoch = cmd_output.split()[-1]
+
+            test_file_suffix = "_{}".format(i)
+            i += 1
+
+            # Run ior command.
+            try:
+                self.run_ior_with_pool(
+                    timeout=200, stop_dfuse=False, create_pool=False,
+                    create_cont=False, test_file_suffix=test_file_suffix)
+            except TestFail:
+                self.log.info("ior command failed!")
+
+            # Get epoch of the created file.
+            creation_epoch = -1
+            cmd_output = ""
+            # As in date command, run stat command in the client node.
+            testfile_path = testfile_path_base + test_file_suffix
+            task = run_task(
+                self.hostlist_clients, "stat -c%Z {}".format(testfile_path))
+            for output, _ in task.iter_buffers():
+                cmd_output = "\n".join(
+                    [line.decode("utf-8") for line in output])
+
+            # The output may contain some warning messages, so use split to get
+            # the value we want.
+            creation_epoch = cmd_output.split()[-1]
+
+            # Calculate the epoch difference between the creation time and the
+            # value in the file metadata. They're usually 5 to 10 sec apart.
+            creation_epoch_int = int(creation_epoch)
+            current_epoch_int = int(current_epoch)
+            diff_epoch = creation_epoch_int - current_epoch_int
+            if diff_epoch > 20:
+                msg = "Unexpected creation time! Expected = {}; Actual = {}"
+                error_list.append(
+                    msg.format(current_epoch_int, creation_epoch_int))
+
+            # 2. Verify file size.
+            # Get file size.
+            file_size = get_remote_file_size(
+                self.hostlist_clients[0], testfile_path)
+
+            # Adjust the file size and verify that it matches the expected size.
+            expected_size = block_size[:-1]
+            # Obtained size is in byte, so convert it to MB.
+            file_size_adjusted = file_size / 1024 / 1024
+            if int(expected_size) != file_size_adjusted:
+                msg = "Unexpected file size! Expected = {}; Actual = {}"
+                error_list.append(
+                    msg.format(int(expected_size), file_size_adjusted))
+
+        # Manually stop dfuse because we set stop_dfuse=False in
+        # run_ior_with_pool.
+        self.stop_dfuse()
```

Comment:
This can be removed as stopping dfuse is performed as part of `tearDown()` through the `DfuseTestBase.stop_job_managers()` method.